tests/cuda_test/test_kernel_semantic.py

```diff
@@ -121,4 +121,126 @@ def kernel_call():
     # Check that the error is correct
     error_info = [*errors.error_info_map.values()][0][0]
     assert error_info.symbol.func  == 'kernel_call'
-    assert UNVALID_KERNEL_CALL_TP_BLOCK == error_info.message
+    assert INVALID_KERNEL_CALL_TP_BLOCK == error_info.message
+
+@pytest.mark.parametrize( 'language', [
+        pytest.param("ccuda", marks = pytest.mark.ccuda)
+    ]
+)
+def test_invalid_kernel_config(language):
+    def invalid_kernel_config():
+        @kernel
+        def kernel_call():
+            pass
+        kernel_call[1.0, 2.0]()
+    # Initialize singleton that stores Pyccel errors
+    errors = Errors()
+
+    # epyccel should raise an Exception
+    with pytest.raises(PyccelSemanticError):
+        epyccel(invalid_kernel_config, language=language)
+
+    # Check that we got exactly 2 Pyccel error
+    assert errors.has_errors()
+    assert errors.num_messages() == 2
+
+    # Check that the errors are correct
+    error_info = [*errors.error_info_map.values()][0]
+    assert error_info[0].symbol.func == 'kernel_call'
+    assert error_info[1].symbol.func == 'kernel_call'
+    assert INVALID_KERNEL_CALL_BP_GRID == error_info[0].message
+    assert INVALID_KERNEL_CALL_TP_BLOCK == error_info[1].message
```

Comment:
isn't these error checked separately in the below tests?

Reply:
Yes. Actually, I wasn't sure what caused the checker coverage test to fail (untested lines) so I thought that I should add this case where both of the parameters failed.

Reply:
The coverage problem is happening because the execution of the cuda tests is run separately to the test which tells the coverage what code ran. This was discussed in our last meeting. It is fixed in #1291

Reply:
Thank you for bringing this to my attention. I have already understood this during our previous meeting, so I will go ahead and remove this case as it is no longer necessary.

Reply:
Fixed f33701bcd3d1507b53b6c8127205a6ba06b54ae8